code/modules/hydroponics/grown.dm

```diff
@@ -499,8 +499,8 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/icepepper
 	seed = /obj/item/seeds/icepepperseed
-	name = "ice-pepper"
-	desc = "It's a mutant strain of chili"
+	name = "chilly pepper"
+	desc = "It's cold to the touch let alone to your taste buds."

```

Comment:
I just realized the description is stupid because your taste buds have nothing to do with "tasting" how cold food is :^)

Gonna change it to more closely resemble the chili pepper description.